### ISSUE TITLE
Does not use net mask

### ISSUE THREAD

**Issue by cancan101** [jdebetaz/node-icloud-private-relay#1]
The code just uses the address portion and ignores the mask which means the full range of IPs for a given line are not handled:
https://github.com/jdebetaz/node-icloud-private-relay/blob/65b4338ad770134ff919d07d32186c93f4e149de/src/index.js#L11

**Comment by cancan101** [comment]
This code works (uses a cidr library):
```js
const checkIp = "146.75.253.254"

const axios = require('axios')
const IPCIDR = require("ip-cidr");

const response = await axios.get("https://mask-api.icloud.com/egress-ip-ranges.csv")
const lines = response.data.split('\n')
const data = lines.map((line) => {
    let l = line.split(',')

    return { cidr: new IPCIDR(l[0]), countryCode: l[1], language: l[2], location: l[3]}
    
})

const ret = data.find(x => x.cidr.contains(checkIp))
```

**Comment by jdebetaz** [comment]
Hello, 

Thank you for your interest in this project, and for your participation.

The issue has just been corrected in the new version ([v1.2.0](https://github.com/jdebetaz/node-icloud-private-relay/releases/tag/v1.2.0))

Kind reagards,
Jonathan
